core/flow/tc_default_test.go

```diff
@@ -0,0 +1,90 @@
+package flow
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/sentinel-group/sentinel-golang/core/base"
```

Comment:
Please import "github.com/alibaba/sentinel-golang/core/base"
the module "github.com/sentinel-group/sentinel-golang/" had been deprecated.

Reply:
@louyuting other test file use "github.com/sentinel-group/sentinel-golang" not "github.com/alibaba/sentinel-golang"

??

Reply:
local modify and test.....
![image](https://user-images.githubusercontent.com/6278792/75438600-2d1a8a00-5993-11ea-908f-c9044bf8e5f5.png)
go.mod modified....

Reply:
Yes, we had update the module name as "github.com/alibaba/sentinel-golang".
You should stop using "github.com/sentinel-group/sentinel-golang" in your code